core/src/test/scala/com/salesforce/op/stages/impl/insights/RecordInsightsLOCOTest.scala

```diff
@@ -202,7 +202,8 @@ class RecordInsightsLOCOTest extends FlatSpec with TestSparkContext with RecordI
     }
   }
 
-  it should "return the most predictive features for data generated with a strong relation to the label" in {
+  describe("data strongly related to label. ") {
```

Comment:
This description doesn't seem to match the following section. Can you add something more descriptive?